### PR TITLE
Multipleanswers

### PR DESCRIPTION
**Pull request by candacejwelch**
Able to handle multiple answers as an option.

### CODE REVIEW

File: KataBase/Question.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@ public class Question
         /// <summary>
         /// The desired text
         /// </summary>
-        public string DesiredAnswer { get; set; }
+        public string[] DesiredAnswer { get; set; }
```
Comment (codytwinton) [inline KataBase/Question.cs]: I'd like to see this renamed to `DesiredAnswers`, as this is an array with multiple values stored within itself. Plural naming is more readable.

File: KataBase/RandomKata.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ public class RandomKata : Kata
         
         public override Question AnswerQuestionAndGetNext(string answer)
         {
-            var isCorrect = _currentQuestion.DesiredAnswer == answer;
+            var isCorrect = Equals(_currentQuestion.DesiredAnswer, answer);
```
Comment (codytwinton) [inline KataBase/RandomKata.cs]: Change this to `_currentQuestion.DesiredAnswer.Contains(answer);`

File: KataBase/SequencedKata.cs [hunk 3]
```diff
@@ -11,37 +11,49 @@ public SequencedKata(params Question[] questions) : base(questions)
...
+        
+        /// <summary>
+        /// Answers the current question, records the grade and returns the next question.
+        /// </summary>
+        /// <param name="answer"></param>
+        /// <returns></returns>
         public override Question AnswerQuestionAndGetNext(string answer)
         {
-            var isCorrect = Questions[_index].DesiredAnswer == answer;
+            var isCorrect = false;
```
Comment (codytwinton) [inline KataBase/SequencedKata.cs]: Change this to use `System.Linq` method: `Contains()`

**Comment by candacejwelch** [comment]
Refactored desiredAnswers class and changed the foreach loop to use .Contain() instead

**Pull request merged**